src/vtt-segment-loader.js

```diff
@@ -1006,44 +972,166 @@ export default class SegmentLoader extends videojs.EventTarget {
     let segmentInfo = this.pendingSegment_;
     let segment = segmentInfo.segment;
 
-    this.syncController_.probeSegmentInfo(segmentInfo);
+    // Make sure that vttjs has loaded, otherwise, wait till it finished loading
+    if (typeof window.WebVTT !== 'function' &&
+        this.subtitlesTrack_ &&
+        this.subtitlesTrack_.tech_) {
+
+      const loadHandler = () => {
+        this.subtitlesTrack_.tech_.off('vttjsloaded', loadHandler);
+        this.handleSegment_();
+      };
+
+      this.state = 'WAITING_ON_VTTJS';
```

Comment:
This is basically the same as aborting while waiting in the `DECRYPTING` stage, the callback will call `handleSegment_` again and if it was aborted while we were waiting, the first `if` block will reset the loader to `READY`